trustpinning/certs.go

```diff
@@ -161,6 +161,7 @@ func ValidateRoot(prevRoot *data.SignedRoot, root *data.Signed, gun string, trus
 		logrus.Debugf("failed to verify TUF data for: %s, %v", gun, err)
 		return nil, &ErrValidationFail{Reason: "failed to validate integrity of roots"}
 	}
+	signedRoot.Signatures = root.Signatures
```

Comment:
I think it should be fine for all roles except root. It was already fine for snapshot and timestamp and per this PR is fixed for targets / delegations. #800 should fix it for root as well